### DISCUSSION TITLE
demo-avatar.png is oversized

### DISCUSSION THREAD

**Discussion by nohuhu** [realworld-apps/realworld#861] [General]
User Gerome has a userpic set to `demo-avatar.png` which is 1536x1536 pixels and clocks in at 263kb. This affects non-authenticated page load significantly and is pointed out in Lighthouse audit.

Is there a chance that this is a bug that can be corrected? The actual size of avatar pics used in the app is 32x32 px.

**Comment by geromegrignon** [comment]
Hey, thanks for raising such an issue.
I fixed it.

**Reply by nohuhu** [reply]
Thanks Gerome! Much better now.
